initializr-web/src/test/resources/metadata/test-default-2.0.0-ssl.json

```diff
@@ -149,6 +149,20 @@
       }
     ]
   },
+  "configurationFileFormat": {
```

Comment:
This changes the metadata v2.0.0 format. I think we should add a new metadata version v2.3 which adds the `configurationFileFormat` field. You can take a look at `InitializrMetadataV22JsonMapper` and `InitializrMetadataV21JsonMapper` for that.

Reply:
I reverted the changes to `InitializrMetadataV2JsonMapper` and added `InitializrMetadataV23JsonMapper`.